pdfmap/pdfmaze.py

```diff
@@ -21,50 +21,111 @@ class PDFMaze:
     def _parse_objs(
         self,
         lt_objs,
-        confidence: Optional[Real] = None
+        confidence: Optional[Real] = None,
+        split_words: bool = False,
+        key_split_chars: list = [' ']
     ) -> Iterable[TextBox]:
         return itertools.chain.from_iterable(
-            self._parse_obj(obj, confidence=confidence)
+            self._parse_obj(
+                obj, 
+                confidence=confidence, 
+                split_words=split_words, 
+                key_split_chars=key_split_chars
+                )
             for obj in lt_objs
         )
 
     def _parse_obj(
             self,
             obj,
-            confidence: Optional[Real] = None
+            confidence: Optional[Real] = None,
+            split_words: bool = False,
+            key_split_chars: list = [' ']
```

Comment:
You can be more specific here. `key_split_chars` can be any iterable of strings (`tuple`s and `list`s are possible, for instance). So please `from typing import Iterable`, and then make this change:
```suggestion
            key_split_chars: Iterable[str] = (' ',)
```
Note that I also changed the default argument from a list `[' ']` to a tuple `(' ',)`. This is because [you should avoid default mutable arguments](https://stackoverflow.com/q/1132941/5811400).
I also suggest changing this everywhere this appears.